src/MatrixTypes.f90

```diff
@@ -32,6 +32,7 @@
 !>  - @ref BLAS2 "BLAS2": @copybrief BLAS2
 !>  - @ref BLAS3 "BLAS3": @copybrief BLAS3
 !>
+!> TIBWSFB: This documentation doesnt match the current interface
```

Comment:
hahaha don't look at me; that was broken when we went from passing an int for size to using the parameter lists. I'm still angling for init routines that use a struct for passing valid params, rather than a full-fledged parameter list, so I'm not itching to re-write the documentation twice.